_includes/components/slider.html

```diff
@@ -1,27 +1,30 @@
-{% assign event = include.event %}
-{% assign impressions = nil | arrayify %}
-{% for file in site.static_files %}
-  {% assign isImpression = file.path | startswith: "/resources/images/impressions/" %}
-  {% if isImpression %}
-    {% assign fileArray = file | arrayify %}
-    {% assign impressions = impressions | concat: fileArray %}
-  {% endif %}
-{% endfor %}
-{% if impressions.first %}
+{% capture imageQuery %}
+  SELECT ?image ?description ?link
+  WHERE {
+    ?image rdf:type foaf:Image ;
+      rdf:value ?link ;
+      dct:description ?description .
+    FILTER(langMatches(lang(?description), "{{ site.language }}"))
+    #FILTER(strstarts(?link, "/resources/images/impressions/"))
```

Comment:
Can you remove this line?

Reply:
this filter should be back again. so we wouldn't need a meaningful property. just forgot it during debugging.